src/main/java/org/example/Main.java

```diff
@@ -26,27 +25,7 @@ public static void main(String[] args) {
                 System.out.println("Enter second number: ");
                 int num2 = s.nextInt();
 
-                switch (operation) {
-                    case 1:
-                        System.out.println("Result: " + (num1+num2));
-                        break;
-                    case 2:
-                        System.out.println("Result: " + (num1-num2));
-                        break;
-                    case 3:
-                        System.out.println("Result: " + (num1*num2));
-                        break;
-                    case 4:
-                        if (num2 != 0) {
-                            System.out.println("Result: Quotient: " + ((double) num1/num2));
-                            System.out.println("Remainder: " + ((double)num1%num2));
-                        } else {
-                            System.out.println("Error: Division by zero.");
-                        }
-                        break;
-                    default:
-                        System.out.println("Enter a valid operation number.");
-                }
+                performOperation(operation,num1,num2);
 
                 System.out.println("Do you want to continue? (yes/no): ");
                 String yesno = s.next().toLowerCase();
```

Comment:
Close the scanner object for better practice
User s.close() for closing it

Reply:
added anna